FBTweak/FBTweakShakeWindow.m

```diff
@@ -86,6 +79,36 @@ - (void)_presentTweaks
   }
 }
 
+- (FBTweakViewController * _Nullable)findTweakViewController {
+  UIViewController *visibleViewController = self.rootViewController;
+  while (visibleViewController.presentedViewController != nil) {
+    visibleViewController = visibleViewController.presentedViewController;
```

Comment:
Same for: `visibleViewController.presentedViewController`.

Reply:
right, but it's handled correctly